package.json

```diff
@@ -103,18 +36,26 @@
   },
   "size-limit": [
     {
-      "path": "index.js",
-      "limit": "1500 B",
-      "ignore": [
-        "./react-deps.js"
-      ]
+      "path": "packages/wouter/dist/index.js",
+      "limit": "2000 B",
```

Comment:
Limits are increased, may I ask why?

Reply:
to make `size-limit` happy)

I included `react-deps.js` to bundle-size test and it not only increases the size but causes some deoptimizations i guess:

- https://github.com/molefrog/wouter/blob/e5544caa1a9059ff24da4fee4893021068541f34/packages/wouter/src/react-deps.js#L4-L9

this code can be deleted and included in final size tests

Reply:
it should be 

```js
import { 
   useEffect, 
   useLayoutEffect, 
   useRef, 
   useInsertionEffect as useBuiltinInsertionEffect, 
 } from 'react'; 
```

to fix all three-shaky stuff

Reply:
but it can break some old react versions that doesn't provide `useInsertionEffect`